Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -43,6 +44,8 @@
      * - 'use' remove blank lines between 'use' import statements
      *
      * @param string[]|null $configuration
+     *
+     * @throws InvalidFixerConfigurationException
```

Comment:
We don't use that annotation anywhere.
Introduce it everywhere or nowhere. Use them in 1% of places is just a noise.